model/prelude.sail

```diff
@@ -221,6 +222,20 @@ function log2(n) = {
   result
 }
 
+/* Carry-less multiply. */
+val carryless_mult : forall 'n, 'n > 0. (bits('n), bits('n)) -> bits(2 * 'n)
+function carryless_mult(a, b) = {
+  var result : bits(2 * 'n) = zeros();
+  foreach (i from 0 to ('n - 1)) {
+    if   a[i] == bitone
+    then result = result ^ (zero_extend(b) << i);
+  };
+  result
+}
+
+infix 7 *_c
```

Comment:
Do we really need this? It's convenient, but when looking at the sail code integrated into the main I would find this much more confusing than a "carryless_mul(a, b)" function call.

with `*_c` I'd have no idea what it is, whereas a well-named function makes it obvious.